profiles/package.mask

```diff
@@ -32,6 +32,19 @@
 
 #--- END OF EXAMPLES ---
 
+# Craig Andrews <candrews@integralblue.com> (2021-07-10)
+# Masked until 2021-11-01
+# This release adds enforcement of the Taproot protocol change to the Bitcoin
+# rules, beginning in November. Protocol changes require user consent to be
+# effective, and if enforced inconsistently within the community may compromise
+# your security or others! If you do not know what you are doing, learn more
+# before November. (You must make a decision either way - simply not upgrading
+# is insecure in all scenarios.)
+# To learn more, see https://bitcointaproot.cc
+>=net-p2p/bitcoin-cli-0.21.1
```

Comment:
No reason to mask bitcoin-cli. It's just a JSON-RPC client.

Reply:
Good point